### DISCUSSION TITLE
Nominatim SAAS

### DISCUSSION THREAD

**Discussion by StarGPS** [osm-search/Nominatim#3183] [Ideas]
Hello,

I've been searching online for solutions to set up Nominatim as a service, like LocationIQ for example, and I couldn't find an answer.
Could you please me help to understand how approach this project what requirements are needed, and if there are any ready solutions to use to accomplish this goal?

Thanks any advance for your answers!

**Comment by lonvia** [comment]
There are instructions to set up a website in the [Deploy](https://nominatim.org/release-docs/latest/admin/Deployment/) section of the documentation. Some folkes have also created a [docker image](https://github.com/mediagis/nominatim-docker/)  to make setup somewhat easier.

Other than that, you will need to find a hosting service and follow their specific instructions.
